src/azul/service/manifest_service.py

```diff
@@ -705,7 +705,9 @@ def _presign_manifest(self,
         if not generator_cls.use_content_disposition_file_name:
             file_name = None
         object_key = generator_cls.s3_object_key(manifest_key)
-        presigned_url = self.storage_service.get_presigned_url(object_key, file_name)
+        role_arn = config.lambda_role_arn('service')
+        with aws.assumed_role_credentials(role_arn=role_arn):
```

Comment:
My personal preference would be to inline this section, but I leave it at your discretion.
```suggestion
        with aws.assumed_role_credentials(config.lambda_role_arn('service')):
```

Do consider that the argument to this method is conventionally treated as a positional, referencing https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#125method-and-function-arguments the first bulleted item in the section permits the omission of the `role_arn` keyword, however you may want to consider enforcing the last bulleted item.